Github Restful API is down

- Looks like the Github Restful API is down, which has nothing to do with me. See https://github.com/linghengqian/hive-server2-jdbc-driver/actions/runs/12973864917/job/36183041426 and https://github.com/linghengqian/hive-server2-jdbc-driver/pull/18 .
```shell
WARNING Failed to create review comment, fallback to a plain comment: POST https://api.github.com/repos/linghengqian/hive-server2-jdbc-driver/pulls/18/reviews: 403 Resource not accessible by integration [] 
WARNING Failed to create review comments POST https://api.github.com/repos/linghengqian/hive-server2-jdbc-driver/pulls/18/reviews: 403 Resource not accessible by integration [] 
ERROR the following files don't have a valid license header: 
.github/workflows/test.yml
.gitignore
.licenserc.yaml
hive-server2-jdbc-driver-thin/pom.xml
hive-server2-jdbc-driver-thin/src/main/resources/META-INF/native-image/io.github.linghengqian/hive-server2-jdbc-driver-thin/native-image.properties
hive-server2-jdbc-driver-thin/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/thin/ClassExistTest.java
hive-server2-jdbc-driver-thin/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/thin/StandaloneMetastoreTest.java
hive-server2-jdbc-driver-thin/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/thin/ThinTest.java
hive-server2-jdbc-driver-thin/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/thin/ZookeeperServiceDiscoveryTest.java
hive-server2-jdbc-driver-uber/pom.xml
hive-server2-jdbc-driver-uber/src/main/resources/META-INF/native-image/io.github.linghengqian/hive-server2-jdbc-driver-uber/native-image.properties
hive-server2-jdbc-driver-uber/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/uber/ClassExistTest.java
hive-server2-jdbc-driver-uber/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/uber/StandaloneMetastoreTest.java
hive-server2-jdbc-driver-uber/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/uber/UberTest.java
hive-server2-jdbc-driver-uber/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/uber/ZookeeperServiceDiscoveryTest.java
pom.xml 
ERROR one or more files does not have a valid license header 
Error: Process completed with exit code 1.
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -1,4 +1,4 @@
-# Copyright 2024 Qiheng He
+# Copyright 2025 Qiheng He
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.

--- a/.gitignore
+++ b/.gitignore
@@ -1,4 +1,4 @@
-# Copyright 2024 Qiheng He
+# Copyright 2025 Qiheng He
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.

--- a/.licenserc.yaml
+++ b/.licenserc.yaml
@@ -1,4 +1,4 @@
-# Copyright 2024 Qiheng He
+# Copyright 2025 Qiheng He
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.

--- a/NOTICE
+++ b/NOTICE
@@ -1,4 +1,4 @@
-Copyright 2024 Qiheng He
+Copyright 2025 Qiheng He
 
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.

--- a/hive-server2-jdbc-driver-thin/pom.xml
+++ b/hive-server2-jdbc-driver-thin/pom.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright 2024 Qiheng He
+  ~ Copyright 2025 Qiheng He
   ~
   ~ Licensed under the Apache License, Version 2.0 (the "License");
   ~ you may not use this file except in compliance with the License.

--- a/hive-server2-jdbc-driver-thin/src/main/resources/META-INF/native-image/io.github.linghengqian/hive-server2-jdbc-driver-thin/native-image.properties
+++ b/hive-server2-jdbc-driver-thin/src/main/resources/META-INF/native-image/io.github.linghengqian/hive-server2-jdbc-driver-thin/native-image.properties
@@ -1,4 +1,4 @@
-# Copyright 2024 Qiheng He
+# Copyright 2025 Qiheng He
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.

--- a/hive-server2-jdbc-driver-thin/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/thin/ClassExistTest.java
+++ b/hive-server2-jdbc-driver-thin/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/thin/ClassExistTest.java
@@ -1,5 +1,5 @@
 /*
- * Copyright 2024 Qiheng He
+ * Copyright 2025 Qiheng He
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/hive-server2-jdbc-driver-thin/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/thin/StandaloneMetastoreTest.java
+++ b/hive-server2-jdbc-driver-thin/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/thin/StandaloneMetastoreTest.java
@@ -1,5 +1,5 @@
 /*
- * Copyright 2024 Qiheng He
+ * Copyright 2025 Qiheng He
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/hive-server2-jdbc-driver-thin/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/thin/ThinTest.java
+++ b/hive-server2-jdbc-driver-thin/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/thin/ThinTest.java
@@ -1,5 +1,5 @@
 /*
- * Copyright 2024 Qiheng He
+ * Copyright 2025 Qiheng He
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/hive-server2-jdbc-driver-thin/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/thin/ZookeeperServiceDiscoveryTest.java
+++ b/hive-server2-jdbc-driver-thin/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/thin/ZookeeperServiceDiscoveryTest.java
@@ -1,5 +1,5 @@
 /*
- * Copyright 2024 Qiheng He
+ * Copyright 2025 Qiheng He
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/hive-server2-jdbc-driver-uber/pom.xml
+++ b/hive-server2-jdbc-driver-uber/pom.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright 2024 Qiheng He
+  ~ Copyright 2025 Qiheng He
   ~
   ~ Licensed under the Apache License, Version 2.0 (the "License");
   ~ you may not use this file except in compliance with the License.

--- a/hive-server2-jdbc-driver-uber/src/main/resources/META-INF/native-image/io.github.linghengqian/hive-server2-jdbc-driver-uber/native-image.properties
+++ b/hive-server2-jdbc-driver-uber/src/main/resources/META-INF/native-image/io.github.linghengqian/hive-server2-jdbc-driver-uber/native-image.properties
@@ -1,4 +1,4 @@
-# Copyright 2024 Qiheng He
+# Copyright 2025 Qiheng He
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.

--- a/hive-server2-jdbc-driver-uber/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/uber/ClassExistTest.java
+++ b/hive-server2-jdbc-driver-uber/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/uber/ClassExistTest.java
@@ -1,5 +1,5 @@
 /*
- * Copyright 2024 Qiheng He
+ * Copyright 2025 Qiheng He
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/hive-server2-jdbc-driver-uber/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/uber/StandaloneMetastoreTest.java
+++ b/hive-server2-jdbc-driver-uber/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/uber/StandaloneMetastoreTest.java
@@ -1,5 +1,5 @@
 /*
- * Copyright 2024 Qiheng He
+ * Copyright 2025 Qiheng He
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/hive-server2-jdbc-driver-uber/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/uber/UberTest.java
+++ b/hive-server2-jdbc-driver-uber/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/uber/UberTest.java
@@ -1,5 +1,5 @@
 /*
- * Copyright 2024 Qiheng He
+ * Copyright 2025 Qiheng He
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/hive-server2-jdbc-driver-uber/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/uber/ZookeeperServiceDiscoveryTest.java
+++ b/hive-server2-jdbc-driver-uber/src/test/java/io/github/linghengqian/hive/server2/jdbc/driver/uber/ZookeeperServiceDiscoveryTest.java
@@ -1,5 +1,5 @@
 /*
- * Copyright 2024 Qiheng He
+ * Copyright 2025 Qiheng He
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/pom.xml
+++ b/pom.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright 2024 Qiheng He
+  ~ Copyright 2025 Qiheng He
   ~
   ~ Licensed under the Apache License, Version 2.0 (the "License");
   ~ you may not use this file except in compliance with the License.

